elytron/WFLY-20037-19496_promote_oidc_authentication_request_scope_to_community.adoc

```diff
@@ -0,0 +1,119 @@
+---
+categories:
+ - elytron
+ - security
+stability-level: community
+issue: https://github.com/wildfly/wildfly-proposals/issues/720
+feature-team:
+ developer: pedro-hos
+ sme:
+  - darranl
+ outside-perspective:
+  - bstansberry
+promotes:
+ - https://issues.redhat.com/browse/WFLY-20037
+ - https://issues.redhat.com/browse/WFLY-19496
+promoted-by:
+---
+
+= Promote OIDC Authentication with Request and Scope parameters to community stability
+:author:            Pedro Silva
+:email:             pesilva@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+OpenID Connect is an authentication mechanism that builds on OAuth 2.0 and allows a user to login to a web application using credentials established by an OpenID provider.
+
+**OIDC whith Additional Scope Values**
+
+WildFly 32 includes the https://docs.wildfly.org/wildfly-proposals/elytron/WFLY-16532-additional-scope-for-auth-request.html[ability to add additional scope values when securing applications using OpenID Connect (OIDC)]. It has now had four releases of bake at preview level and should be promoted to community.
+
+**OIDC whith Request Parameters**
+
+WildFly 33 https://docs.wildfly.org/wildfly-proposals/elytron/WFLY-17143-request-uri-parameters.html[includes the ability to send the request object as a Json Web Token (JWT) when securing an application using OIDC]. The feature also includes the ability to sign and/or encrypt the JWT for added security. It has now had three releases of bake at preview level and should be promoted to community.
+
+=== User Stories
+
+* I'm a developer who wants to use the OpenID Connect to login to my web application using credentials established by an OpenID provider, and I need to use aditional scope values and request rarameters on my application. This is possible using Wildfly with `preview` stability, but I need it at `community` stability.
+
+== Issue Metadata
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-20037
+* https://issues.redhat.com/browse/WFLY-19496
+
+=== Affected Projects or Components
+
+* https://github.com/wildfly/wildfly
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+
+* Traditional standalone server (unzipped or provisioned by Galleon)
+* Managed domain
+* OpenShift Source-to-Image (S2I)
+* Bootable jar
+
+== Requirements
+
+* The OIDC request and aditional scope parameters will be available when the server is running at `community` stability.
```

Comment:
Could you please list exhaustively the attributes and the resources that will actually be promoted to community? eg

* [`scope`](https://docs.wildfly.org/36/wildscribe/subsystem/elytron-oidc-client/secure-deployment/index.html#attr-scope) attribute of the `provider`, `realm`, `secure-deployment`, and `secure-server` resources